dashboard/client/src/pages/overview/cards/NodeCountCard.tsx

```diff
@@ -33,9 +34,10 @@ const useStyles = makeStyles((theme) =>
 
 type NodeCountCardProps = {
   className?: string;
+  job: UnifiedJob;
 };
```

Comment:
this component is also used in teh Overview page which is not scoped to a single job. Let's keep this component to be generic. Let's take in optional from and to parameters here, and pass it in from JobDetail as job.start_time and job.end_time.

Also, do you know if job start_time and job end_time is in the right timestamp format for grafana?